### ISSUE TITLE
initialScrollToIndex does not work with fewer than 11 items

### ISSUE THREAD

**Issue by pmkemper** [rdhox/react-native-smooth-picker#26]
The issue is easy to replicate: 
In the example in this repo, remove all cities after "Amsterdam". 
initiateScrollToIndex no longer works now. It is not until you scroll that the list jumps to the correct index.

This is how the SmoothPicker initially renders with the below code:
<img width="373" alt="Screen Shot 2021-03-17 at 10 02 15" src="https://user-images.githubusercontent.com/3207921/111441767-ed425400-8707-11eb-9979-e6e480ec6afb.png">

```
const dataCity = [
  "Paris",
  "Berlin",
  "Lisbonne",
  "Budapest",
  "Londres",
  "Prague",
  "Rome",
  "Barcelone",
  "Amsterdam",
];

const opacities = {
  0: 1,
  1: 1,
  2: 0.6,
  3: 0.3,
  4: 0.1,
};
const sizeText = {
  0: 20,
  1: 15,
  2: 10,
};

const Item = React.memo(({ opacity, selected, vertical, fontSize, name }) => {
  return (
    <View
      style={[
        styles.OptionWrapper,
        {
          opacity,
          borderColor: selected ? "#ABC9AF" : "transparent",
          width: vertical ? 190 : "auto",
        },
      ]}>
      <Text style={{ fontSize }}>{name}</Text>
    </View>
  );
});

const ItemToRender = ({ item, index }, indexSelected, vertical) => {
  const selected = index === indexSelected;
  const gap = Math.abs(index - indexSelected);

  let opacity = opacities[gap];
  if (gap > 3) {
    opacity = opacities[4];
  }
  let fontSize = sizeText[gap];
  if (gap > 1) {
    fontSize = sizeText[2];
  }

  return (
    <Item
      opacity={opacity}
      selected={selected}
      vertical={vertical}
      fontSize={fontSize}
      name={item}
    />
  );
};

const Test = () => {
  function handleChange(index) {
    setSelected(index);
    refPicker.current.scrollToIndex({
      animated: false,
      index: index,
      viewOffset: -30,
    });
  }

  const [selected, setSelected] = useState(5);
  const refPicker = useRef(null);
  return (
    <View style={styles.container}>
      <View style={styles.wrapperHorizontal}>
        <SmoothPicker
          initialScrollToIndex={selected}
          refFlatList={refPicker}
          keyExtractor={(_, index) => index.toString()}
          horizontal={true}
          scrollAnimation
          showsHorizontalScrollIndicator={false}
          data={dataCity}
          renderItem={(option) => ItemToRender(option, selected, false)}
        />
      </View>
      <View style={styles.wrapperVertical}>
        <SmoothPicker
          initialScrollToIndex={selected}
          onScrollToIndexFailed={() => {}}
          keyExtractor={(_, index) => index.toString()}
          showsVerticalScrollIndicator={false}
          data={dataCity}
          scrollAnimation
          onSelected={({ item, index }) => handleChange(index)}
          renderItem={(option) => ItemToRender(option, selected, true)}
          magnet
        />
      </View>
      <View>
        <Text>{`Your selection is ${dataCity[selected]}`}</Text>
      </View>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    paddingTop: 60,
    paddingBottom: 30,
    flex: 1,
    flexDirection: "column",
    justifyContent: "space-evenly",
    alignItems: "center",
    backgroundColor: "#F5FCFF",
  },
  wrapperHorizontal: {
    height: 100,
    justifyContent: "center",
    alignItems: "center",
    margin: "auto",
    color: "black",
  },
  wrapperVertical: {
    width: 250,
    height: 350,
    justifyContent: "center",
    alignItems: "center",
    margin: "auto",
    color: "black",
  },
  OptionWrapper: {
    justifyContent: "center",
    alignItems: "center",
    marginTop: 10,
    marginBottom: 10,
    paddingTop: 10,
    paddingBottom: 10,
    paddingLeft: 30,
    paddingRight: 30,
    height: 50,
    borderWidth: 3,
    borderRadius: 10,
  },
});

export default Test;

```

After digging through the code a bit, I've found that in `_renderItem` in SmoothPicker.js, when there are 11 or more items, `this._alignAfterMount()` does run (in the if statement below), while if there are fewer items, it never runs. 

So, for a reason I don't understand, when there are 11 or more items, the if/else statement below is run through data.length+1 times, but it is run through only data.length times when there are fewer than 11 items?

```
          if (this.countItems === data.length) {
            this.countItems = 0;
            this._alignAfterMount();
          } else {
            this.countItems = this.countItems + 1;
          }
```

**Comment by zpv** [comment]
👍 Still an issue

**Comment by rdhox** [comment]
Should be fix with PR #29 , new version 1.1.4 deployed on npm.
